ganga/GangaCore/Runtime/bootstrap.py

```diff
@@ -722,7 +722,7 @@ def _versionsort(s, p=re.compile(r'^(\d+)-(\d+)-*(\d*)')):
                     return int(m.group(1)), int(m.group(2)), int(m.group(3))
             if s == 'SVN':
                 return 'SVN'
-            return None
+            return 0, 0, 0
```

Comment:
In case you are wondering - you can sort a list of `None` now so something has to be returned.